### PR TITLE
7.4.0 release

### PR DESCRIPTION
**Pull request by HuFangshuai**
Support for Inline Ads(Custome size ads).
VungleBanner replaced with VungleBannerView.

### CODE REVIEW

File: Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java [hunk 1]
```diff
@@ -411,19 +411,44 @@ private void updateUserPrivacySettings(final MaxAdapterParameters parameters)
...
+        Object widthObj = localExtraParameters.get( "adaptive_banner_width" );
+        if (widthObj instanceof Integer) {
+            customWidth = (int) widthObj;
+        }
+        int customHeight = -1;
+        Object heightObj = localExtraParameters.get( "adaptive_banner_height" );
+        if (heightObj instanceof Integer) {
+            customHeight = (int) heightObj;
+        }
+        if ( !isAdaptiveBanner && customWidth != -1 && customHeight != -1 ) {
```
Comment (Str4tos) [inline Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java:437]: Hi. I think I found a typo.
```suggestion
        if ( isAdaptiveBanner && customWidth != -1 && customHeight != -1 ) {
```

File: Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java [hunk 2]
```diff
@@ -521,8 +546,14 @@ private Bundle maybeCreateExtraInfoBundle(final BaseAd baseAd)
 
         if ( TextUtils.isEmpty( creativeId ) ) return null;
```
Comment (Str4tos) [inline Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java:547]: Hi. Here’s another observation: if the `creativeId` is empty, the banner size won't be provided in the `extraInfo`.

File: Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java (hunk 1 above)
Comment (HuFangshuai) [reply Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java:437]: Hi, I know the user guide has announced the `adaptive_banner` with true for adaptive banner and pubs can just only pass `adaptive_banner_width` for it. But for now we do not support adaptive banner, only custom banner size. That means we require both width and height for our inline placement.

File: Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java (hunk 2 above)
Comment (HuFangshuai) [reply Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java]: Nice catch, updated.

File: Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java (hunk 1 above)
Comment (ManojBudumuru-vungle) [reply Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java]: HI @Str4tos Updated the pr to be able to support adaptive and custome sizes.

**Pull request closed without merging**